src/shims/unix/linux/fs/event.rs

```diff
@@ -0,0 +1,39 @@
+use crate::shims::unix::fs::FileDescriptor;
+
+use rustc_const_eval::interpret::InterpResult;
+
+use std::io;
+
+#[derive(Debug)]
+pub struct Event {
+    pub val: u32,
+}
+
+impl FileDescriptor for Event {
+    fn name(&self) -> &'static str {
+        "event"
+    }
+
+    fn dup<'tcx>(&mut self) -> io::Result<Box<dyn FileDescriptor>> {
+        Ok(Box::new(Event { val: self.val }))
+    }
+
+    fn is_tty(&self) -> bool {
+        false
+    }
+
+    fn write<'tcx>(
+        &self,
+        _communicate_allowed: bool,
+        bytes: &[u8],
+    ) -> InterpResult<'tcx, io::Result<usize>> {
+        throw_unsup_format!("cannot write to {}, {:?}", self.name(), bytes);
```

Comment:
Why do we print the `bytes` in this error here but not in the default implementation for `fn write`?